### DISCUSSION TITLE
Input Slot

### DISCUSSION THREAD

**Discussion by Jwellll** [jugeeya/UltimateTrainingModpack#295] [Ideas]
This has already been suggested in another idea but I wanted to elaborate further on what I have in mind.

As was said, this is a feature in street fighter training mode. The reason I believe this would be particularly helpful as part of the training mod pack because it would allow players to train against moves that currently inaccessible with the current training mod. For example, there are many characters that have specific specials which the CPU has no toggle (Sephiroth's Megaflare, Pit's direction influenced arrows, Terry's go-meter specials, etc.)

If it were possible to record a number of inputs within a set timer, the player could achieve practice in an infinitude of scenarios. Of course, there would be a few challenges in accomplishing this I'm sure. The player would first have to select the character who's special they want to practice against, record the inputs, save them to a slot, and then exit to the character select to pick their actual character. If possible you could allow to inputs to be recorded with a second controller while using the "control" feature already included in the vanilla training mode.

I don't want to get too far into the technical side of its implementation, because this is definitely not my expertise, but I just wanted to provide a few ideas if helpful whatsoever. Hope to see this in future updates if possible! Please let me know is there anything I can do to help!

**Comment by jugeeya** [comment]
This is in progress under the name "Input Recording"
